### DISCUSSION TITLE
Docs: How to find relevant documents in the `errors` directory in next.js repo

### DISCUSSION THREAD

**Discussion by char8x** [vercel/next.js#73961] [Help]
### Summary

1. It would be better to be more convenient to query related documents in [errors](https://github.com/vercel/next.js/tree/canary/errors).
2. Provide a sidebar entry for errors-related documents.
3. Errors-related documents could be found in the search input.

Some documentation on error messages is already accessible directly via links, e.g. [Dynamic APIs are Asynchronous](https://nextjs.org/docs/messages/sync-dynamic-apis), [`next.config.js` Loading Error](https://nextjs.org/docs/messages/next-config-error)

**Comment by icyJoseph** [comment]
In the meantime, I think the GitHub queries allow you to do this: repo:vercel/next.js path:/^errors\// , https://github.com/search?q=repo%3Avercel%2Fnext.js+path%3A%2F%5Eerrors%5C%2F%2F+&type=code
